src/stories/Login.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { Button } from "./Button";
+import { TextField } from "./TextField";
+import { Checkbox } from "./Checkbox";
+
+export const Login = () => (
+  <div className="storybook-login">
+    <h1 className="storybook-login__title">Login</h1>
+    <TextField label="Email" placeholder="hello@world.com" />
+    <TextField label="Password" placeholder="password123" />
```

Comment:
I'd actually suggest changing this to a more generic `TextInputField` that also takes in `type`. There are specific Input fields for content like [Email](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) and [Password](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password) that have their own unique attributes. So, while stylistically they may look similar to a basic `type="text"` field, they have unique properties that help handle the content they contain.

A password field for example will automatically hide the actual text values and depending on browser include a "view password" toggle. This also really helps with `autofill` content since the input fields provide context as to what they are for.

Reply:
Ok, that makes sense! So the `TextInput` component could take in an additional (optional) field called `type` that defaults to `"text"`, but the developer could specify `"email"`/`"password"`/some other type that would allow for the correct autofill functionality. If the developer types in an unrecognized value for `type`, then I assume we would have to catch that and default back to the `"text"` value, right? 
I'll try making this fix!

Reply:
Yeah exactly. You could have it take in specific valid `types` like we do for `variant` if you want to be strict. I believe that if `type="<something invalid>"`, html just ignores it so it would default to `text`.